ui/components/app/collectibles-items/collectibles-items.component.js

```diff
@@ -109,15 +149,28 @@ export default function CollectiblesItems({ onAddNFT, onRefreshList }) {
               alignItems={ALIGN_ITEMS.CENTER}
               justifyContent={JUSTIFY_CONTENT.CENTER}
             >
-              <Box justifyContent={JUSTIFY_CONTENT.FLEX_END}>
-                <Button
-                  type="link"
-                  onClick={onRefreshList}
-                  style={{ padding: '4px' }}
-                >
-                  {t('refreshList')}
-                </Button>
-              </Box>
+              {' '}
+              {useCollectibleDetection ? (
+                <Box justifyContent={JUSTIFY_CONTENT.FLEX_END}>
+                  <Button
+                    type="link"
+                    onClick={onRefreshList}
+                    style={{ padding: '4px', fontSize: '16px' }}
+                  >
+                    {t('refreshList')}
+                  </Button>
+                </Box>
+              ) : (
+                <Box justifyContent={JUSTIFY_CONTENT.FLEX_END}>
+                  <Button
+                    type="link"
+                    onClick={onEnableAutoDetect}
+                    style={{ padding: '4px', fontSize: '16px' }}
```

Comment:
Lets avoid using inline styled like above.

Reply:
agreed, let's remove inline styles from this file

Reply:
Agreed! Thanks! Will address in subsequent PR! (I'm noting these requests for a followup ticket!)